### ISSUE TITLE
getting empty test-explorer tab when using vscode-remote-wsl

### ISSUE THREAD

**Issue by ericblade** [hbenl/vscode-test-explorer#100]
I have a project that i'm trying to run in vscode-remote-wsl, but the test tab appears blank.  I've installed all the correct plugins, I believe -- Mocha Test Explorer, and Test Explorer.  The Test Explorer and Mocha Test Explorer output windows are completely empty, as well. 

None of the other output tabs make any mention of the extension as far as I can see.  

Any ideas?

**Comment by hbenl** [comment]
Does it work when you open the folder locally? Is there a repo with which I can reproduce this?

**Comment by ericblade** [comment]
hmmm.  Good question.  It *had* been working in local Windows, but not in WSL.. but at the moment it's not doing anything in either.  I can see it refreshing in the Test Explorer tab when I make changes to the test files, but nothing shows up, and no log information.

**Comment by hbenl** [comment]
Try setting `mochaExplorer.logpanel` to `true`, then you'll get an output channel called "Mocha Explorer Log" that shows what the test adapter is doing.

**Comment by ericblade** [comment]
Thanks, looks like it's some sort of problem in determining what files have tests:

````
[2019-11-21 16:49:21.927] [DEBUG] Looking for test files ["./test/test-*.js"] in d:\src\asset-bundle-service
[2019-11-21 16:49:21.994] [DEBUG] Found test files []
````

Any way to get a better handle on why it thinks none of my test-* files have tests?

**Comment by hbenl** [comment]
try replacing `./test/test-*.js` with `test/test-*.js`

**Comment by ericblade** [comment]
alright, i changed that in my mocha.opts .. now it errors in Windows due to having a Linux binary (expected) in node_modules, and it works in WSL! nice.  Thanks for the assist there. I wonder if the search for test files can be made more robust to handle ./ or at least the extension could provide some information when nothing is found, without enabling special debug options?

**Comment by hbenl** [comment]
Yeah, I was surprised about this too - I'm using a [VS Code API](https://code.visualstudio.com/api/references/vscode-api#RelativePattern) to find the files and would have expected it to work with a glob that starts with `./`.
I'll think about how to handle this situation better.

**Comment by hbenl** [comment]
This is fixed in version 2.2.1 of the Mocha Test Explorer.
